ocs_ci/deployment/deployment.py

```diff
@@ -1268,6 +1268,9 @@ def deploy_lvmo(self):
             constants.TEMPLATE_DEPLOYMENT_DIR_LVMO,
             f"lvm-cluster-{file_version}.yaml",
         )
+
+        if int(lvmo_version_without_period) >= 411:
```

Comment:
can we use  Version.coerce()?

Reply:
If we have already the data available why add more actions? What is the benefit. If the data wasn't here I would do it with Version.corece but since I have already the data from other actions I do here, I don't understand the benefit.

Reply:
My intention is that we will not convert to int but rather use a proper way of comparing versions, semantically